apps/web/src/state/farms/hooks.ts

```diff
@@ -95,38 +91,28 @@ export const usePollFarmsAvgInfo = (activeFarms: (V3FarmWithoutStakedValue | V2F
 
       const addresses = activeFarms.map((farm) => farm.lpAddress?.toLowerCase())
 
-      const rawResult: any | undefined = await multiQuery(
```

Comment:
The original query is v3 only? I see there's v3 client and checking

Reply:
yup